www/docs/en/dev/config_ref/index.md

```diff
@@ -45,207 +45,356 @@ passively copied into various `platforms/` subdirectories, for example:
         app/platforms/blackberry10/www/config.xml
```

Comment:
In above paragraph: "... subdirectories, for example:" should be "... subdirectories. For example:".